docs/app/references/changelog.mdx

```diff
@@ -8,6 +8,30 @@ sidebar_label: Changelog
 
 # Changelog
 
+## 14.1.0
+
+_Released 2/25/2025_
+
+**Features:**
+
+- Firefox versions 135 and up are now automated with [WebDriver BiDi](https://www.w3.org/TR/webdriver-bidi/) instead of [Chrome Devtools Protocol](https://chromedevtools.github.io/devtools-protocol/). Addresses [#30220](https://github.com/cypress-io/cypress/issues/30220).
```

Comment:
```suggestion
- Firefox versions 135 and above are now automated with [WebDriver BiDi](https://www.w3.org/TR/webdriver-bidi/) instead of [Chrome Devtools Protocol](https://chromedevtools.github.io/devtools-protocol/). Addresses [#30220](https://github.com/cypress-io/cypress/issues/30220).
```

For consistency with other places that refer also to higher versions.